pkg/plugin/base/api.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v2
+package base
```

Comment:
IHMO we should keep V2. 
a) at some point we will be able to remove the V2 and all will be V3
b) `In the case of the `v1` and `v2` PROJECT version, a corresponding Plugin version is implied and constant -- `v1` implies the `go.kubebuilder.io/v1` Plugin, and similarly for `v2`.  This is for legacy purposes -- no such implication is made with the `v3` PROJECT version.`

Reply:
Updated base -> v0.